public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php

```diff
@@ -762,6 +631,27 @@ protected function get_stub_posts( $wordcamp, $meta ) {
 		return $posts;
 	}
 
+	/**
+	 * Load the content for a stub from an include file.
+	 *
+	 * @param string $post_type
+	 * @param string $stub_name
+	 *
+	 * @return string
+	 */
+	protected function get_stub_content( $post_type, $stub_name ) {
+		$content   = '';
+		$stub_file = WCPT_DIR . "stubs/$post_type/$stub_name.php";
+
+		if ( is_readable( $stub_file ) ) {
```

Comment:
I'm not sure we need this check here, since it's our own file and inside this same plugin. If one of us changes one of the files and forgot to update calls to it, we'd probably want to see a fatal in our local environments to notify us before we deploy that change.

To me, having this check feels similar to `WordCamp_New_Site::configure_new_site()` doing this:

```php
if ( is_callable( array( $this, 'get_user_or_current_user' ) ) {
	$lead_organizer = $this->get_user_or_current_user( $meta['WordPress.org Username'][0] );
}
```

Having said that, I don't feel strongly about it.

Reply:
I see what you're saying. If we did change something that cause this to fatal, though, it would halt the whole process of setting up a new site, which I think is a lot worse than having a stubbed page with missing content. Maybe there could be an `else` that logs an error if the file isn't readable?

Reply:
Yeah, that's a fair compromise :)